tests/back/Pim/Enrichment/Integration/Command/Category/RemoveCategoriesOrphan/RemoveOrphanCategoriesIntegration.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace AkeneoTest\Pim\Enrichment\Integration\Command\Category\RemoveCategoriesOrphan;
+
+use Akeneo\Category\Domain\Model\Enrichment\Category;
+use Akeneo\Category\Domain\Query\GetCategoryInterface;
+use Akeneo\Test\Integration\Configuration;
+use Akeneo\Test\Integration\TestCase;
+use Doctrine\DBAL\Driver\Connection;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+
+/**
+ * @copyright 2022 Akeneo SAS (http://www.akeneo.com)
```

Comment:
```suggestion
/**
 * @copyright 2023 Akeneo SAS (http://www.akeneo.com)
```

Be careful when copying/pasting :)